extensions/wikia/EmergencyBroadcastSystem/js/EmergencyBroadcastSystem.js

```diff
@@ -1,11 +1,22 @@
-(function ($) {
+( function ($) {
 	'use strict';
 
 	function handleClick( val ) {
-		$.nirvana.postJson('EmergencyBroadcastSystemController', 'saveUserResponse', { val: val });
-		$('.ebs-container').hide();
+		$( '.ebs-container' ).hide();
+		return $.nirvana.postJson( 'EmergencyBroadcastSystemController', 'saveUserResponse', { val: val } );
 	}
 
-	$('.ebs-primary-action').click(function(){ handleClick( 1 ); return false; });
-	$('.ebs-secondary-action').click(function(){ handleClick( 0 ); return false; });
-})(jQuery);
+	$( '.ebs-primary-action' ).click( function(){
+		var href = this.href;
+		var request = handleClick( 1 );
```

Comment:
```
handleClick( 1 ).then( function() {
...
}
```
That way you don't not the request variable.
Btw. using then is sort of better - because it does the same thing (more or less) but it is a promise standard - according to spec.